tests/integration/helpers/cluster.py

```diff
@@ -3200,12 +3200,20 @@ def shutdown(self, kill=True, ignore_fatal=True):
         if fatal_log is not None:
             raise Exception("Fatal messages found: {}".format(fatal_log))
 
-    def pause_container(self, instance_name):
+    def _pause_container(self, instance_name):
         subprocess_check_call(self.base_cmd + ["pause", instance_name])
 
-    def unpause_container(self, instance_name):
+    def _unpause_container(self, instance_name):
         subprocess_check_call(self.base_cmd + ["unpause", instance_name])
 
+    @contextmanager
+    def pause_container(self, instance_name):
```

Comment:
Since it changes the behavior, I think it worth to add the docstring here
```suggestion
    def pause_container(self, instance_name):
        '''Use it as following:
        with cluster.pause_container(name):
            useful_stuff()
        '''
```